src/integrationTest/java/uk/gov/hmcts/reform/userprofileapi/client/UserProfileRequestHandlerTest.java

```diff
@@ -73,7 +72,6 @@ public MvcResult sendGet(MockMvc mockMvc,
         return mockMvc.perform(get(path)
                 .headers(getMultipleAuthHeaders())
                 .contentType(APPLICATION_JSON))
-                .andExpect(status().is(expectedHttpStatus.value()))
```

Comment:
any reason for removing this? Can you point the code where you are really asserting the http status value?